src/bscPlugin/codeActions/XmlFileCodeActionsProcessor.ts

```diff
@@ -18,19 +19,19 @@ export class XmlFileCodeActionsProcessor {
     public process() {
         for (const diagnostic of this.diagnostics) {
             if (diagnostic.code === DiagnosticCodeMap.xmlComponentMissingExtendsAttribute) {
-                this.addMissingExtends();
+                this.addMissingExtends(diagnostic as any);
             }
         }
     }
 
-    private addMissingExtends() {
+    private addMissingExtends(diagnostic: DiagnosticMessageType<'xmlComponentMissingExtendsAttribute'>) {
         const { component } = this.file.parser.ast;
         //inject new attribute after the final attribute, or after the `<component` if there are no attributes
         const pos = (component.attributes[component.attributes.length - 1] ?? component.tag).range.end;
         this.codeActions.push(
             codeActionUtil.createCodeAction({
                 title: `Extend "Group"`,
-                // diagnostics: [diagnostic],
+                diagnostics: [diagnostic],
```

Comment:
We can send diagnostics for these actions now that we are sanitizing them before sending them in the language server.

Reply:
What does the diagnostic give?

Reply:
I don't actually know. I looked through the docs and they don't explain it at all.